<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setOpwidget</class>
 <widget class="QWidget" name="setOpwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="setOpLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SET INSTRUCTION:</string>
   </property>
  </widget>
  <widget class="QLabel" name="formLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(NAME DESTI SOURCEJ SOURCEK)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="indexLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(INDEX)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="indexEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="setOpEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
